lib/core/components/MantraForm.js

```diff
@@ -56,4 +71,21 @@ export default class MantraForm extends MantraComponent {
         const filteredInfo = pathInfo.filter((str) => str);
         return filteredInfo.join('.');
     };
+
+    /**
+     * Verify the validity of the MantraForm constructor params
+     * @param {Object} value - The constructor params
+     */
```

Comment:
As mentioned in previous comments, I would suggest adding a JSDoc `returns` statements that specify that this method may return either an `Error` or `void`.

```suggestion
     * @returns {Error|void} [Here a message of why one of the data-types is returned]
     */
```